web/templates/page/index.html.eex

```diff
@@ -1,36 +1,23 @@
 <div class="jumbotron">
-  <h2><%= gettext "Welcome to %{name}", name: "Phoenix!" %></h2>
-  <p class="lead">A productive web framework that<br />does not compromise speed and maintainability.</p>
+  <h2><%= gettext "Welcome to %{name}", name: "Notox" %></h2>
+  <p class="lead">A lightweight note-taking application <br/> built with Elixir and Phoenix!</p>
 </div>
 
-<div class="row marketing">
-  <div class="col-lg-6">
-    <h4>Resources</h4>
-    <ul>
-      <li>
-        <a href="http://phoenixframework.org/docs/overview">Guides</a>
-      </li>
-      <li>
-        <a href="https://hexdocs.pm/phoenix">Docs</a>
-      </li>
-      <li>
-        <a href="https://github.com/phoenixframework/phoenix">Source</a>
-      </li>
-    </ul>
-  </div>
+  <%= if @current_user do %>
 
-  <div class="col-lg-6">
-    <h4>Help</h4>
-    <ul>
-      <li>
-        <a href="http://groups.google.com/group/phoenix-talk">Mailing list</a>
-      </li>
-      <li>
-        <a href="http://webchat.freenode.net/?channels=elixir-lang">#elixir-lang on freenode IRC</a>
-      </li>
-      <li>
-        <a href="https://twitter.com/elixirphoenix">@elixirphoenix</a>
-      </li>
-    </ul>
-  </div>
-</div>
+    <p>Logged in as <%= link @current_user.username,
+      to: user_path(@conn, :show, @current_user) %></p>
+    <p><%= link "Logout", 
+      to: session_path(@conn, :delete, @current_user), 
+      method: :delete %></p>
+    <p><%= link "Users", 
+      to: user_path(@conn, :index) %></p>
+    <p><%= link "Notes", 
+      to: note_path(@conn, :index) %></p>
+
+  <% else %>
+
+    <p><%= link "Login", to: session_path(@conn, :new) %></p>
```

Comment:
Should these links go in the header bar somewhere? Should the Phoenix logo and "getting started" link be removed?